dev-lang/tcl/tcl-8.6.14.recipe

```diff
@@ -9,24 +9,17 @@ COPYRIGHT="Regents of the University of California, Sun Microsystems, Inc., Scri
 LICENSE="TCL"
 REVISION="1"
 SOURCE_URI="https://sourceforge.net/projects/tcl/files/Tcl/$portVersion/tcl$portVersion-src.tar.gz"
-CHECKSUM_SHA256="43a1fae7412f61ff11de2cfd05d28cfc3a73762f354a417c62370a54e2caf066"
+CHECKSUM_SHA256="5880225babf7954c58d4fb0f5cf6279104ce1cd6aa9b71e9a6322540e1c4de66"
 SOURCE_DIR="tcl$portVersion"
 PATCHES="tcl-$portVersion.patchset"
 
-ARCHITECTURES="all !x86_gcc2"
+ARCHITECTURES="all"
 SECONDARY_ARCHITECTURES="x86"
 
-commandBinDir=$binDir
-commandSuffix=$secondaryArchSuffix
-if [ "$targetArchitecture" = x86_gcc2 ]; then
-	commandSuffix=
-	commandBinDir=$prefix/bin
-fi
-
 PROVIDES="
 	tcl$secondaryArchSuffix = $portVersion compat >= 8.6
-	cmd:tclsh$commandSuffix = $portVersion compat >= 8.6
-	cmd:tclsh8.6$commandSuffix = $portVersion compat >= 8.6
+	cmd:tclsh$secondaryArchSuffix = $portVersion compat >= 8.6
```

Comment:
Should one of the packages (x86_gcc2, x86) not provide tclsh, i.e. remove it, so that we don't have both?

Reply:
Is there any advantige on having a gcc2 package? Otherwise mark it as "?" ?

Reply:
There was one before (version 8.6.11) and it wasn't that much effort to fix, as you can see in the patchset. I was just wondering what to do with tclsh. Another possibility would be to split it and make a separate `_tools` package.

Reply:
Imho I would stick with only providing the binaries for secondary architecture as you suggested in the first comment and remove them for gcc2 (if they aren't required for anything else). But that's just me :)